Makefile.PL

```diff
@@ -48,12 +49,21 @@ my %WriteMakefileArgs = (
     "mro" => 0
   },
   "TEST_REQUIRES" => {
+    "Clone" => 0,
+    "Data::UUID" => 0,
+    "Date::Utility" => 0,
     "ExtUtils::MakeMaker" => 0,
     "File::Spec" => 0,
     "IO::Handle" => 0,
     "IPC::Open3" => 0,
+    "Mojolicious::Lite" => 0,
```

Comment:
Could we avoid bringing this huge dependency only to do tests? We don't need to tests http client library, for this purpose we have tests at http client library. I think better to mock http request rather than to run http server.

Reply:
I see, it was already in this package, but wasn't mentioned in dependencies.  Ok, never mind about my comment.

Reply:
This is the test dependencies.